.circleci/config.yml

```diff
@@ -1,65 +1,28 @@
 version: 2.1
 
 orbs:
-  architect: giantswarm/architect@4.27.0
-
-
-jobs:
-  build:
-    executor: "architect/architect"
-    environment:
-      DOCKER_BUILDKIT: "1"
-
-    steps:
-      - checkout
-
-      - setup_remote_docker:
-          version: 20.10.11
-
-      - architect/push-to-docker:
-          image: "quay.io/giantswarm/docs"
-          tag-latest-branch: "main"
-          username_envar: "QUAY_USERNAME"
-          password_envar: "QUAY_PASSWORD"
-
-      - run:
-          name: Launch container for tests
```

Comment:
@pipo02mix this test gets removed. Should we keep it?

Reply:
I guess we can keep it :) should not be affected by this change